src/index.ts

```diff
@@ -569,35 +576,50 @@ class ForkTsCheckerWebpackPlugin {
   }
 
   private spawnService() {
+    const env: any = {
```

Comment:
Do we need an `any` here? I'll confess to a personal peccadillo: I'm totally prejudiced against `any`. It triggers me 😁

Reply:
Using a stricter type required to cast values so I tried to avoid it, but I don't mind changing that. I pushed a fix 👍